### PR TITLE
feat: open tasks in embedded task view in CLI [DET-7686]

### PR DESCRIPTION
**Pull request by julian-determined-ai**
## Description

feat: open tasks in embedded task view [DET-7686]

JupyterLab notebooks and Tensorboard are not currently being opened in the embedded task view when using the `CLI`. These changes update the `CLI` to use the embedded task view for both.

## Test Plan
Using the CLI:
1. Create a Notebook, it should open in the embedded task view.
2. Open a Notebook, it should open in the embedded task view.
3. Create a Tensorboard, it should open in the embedded task view.
4. Open a Tensorboard, it should open in the embedded task view.

## Commentary (optional)

Currently I cannot open `Tensorboard` locally so I have yet to test that.

## Checklist

- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](../docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

[DET-7686]: https://determinedai.atlassian.net/browse/DET-7686?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: harness/determined/common/api/request.py [hunk 1]
```diff
@@ -41,6 +40,15 @@ def maybe_upgrade_ws_scheme(master_address: str) -> str:
...
         return master_address
 
+def make_interactive_task_url(task_id: str, 
+        service_address: str, 
+        description:str, 
+        resource_pool: str,
+        task_type: str) -> str:
+    wait_path_url = service_address + "/notebooks/{}/events".format(task_id)
+    wait_page_url = "/det/wait/{}/{}?eventUrl={}&serviceAddr={}".format(task_type, task_id,wait_path_url,service_address)
+    notebook_web_url = "/det/interactive/{}/{}/{}/{}/{}".format(task_id,task_type,description,resource_pool,parse.quote_plus(wait_page_url))
```
Comment (julian-determined-ai) [inline harness/determined/common/api/request.py:50]: I am assuming we do not want to hard-code `/det/` here. If that is the case where should the actual value come from?

**Review by mapmeld — APPROVED** [review]
tentatively LGTM -- I'm asking on the halp channel if there is a reason to avoid `/det/` but I am not currently aware of any compatibility issue or a URL generator for it

File: harness/determined/common/api/request.py [hunk 2]
```diff
@@ -42,6 +42,19 @@ def maybe_upgrade_ws_scheme(master_address: str) -> str:
...
 
 
+def make_interactive_task_url(
+    task_id: str, service_address: str, description: str, resource_pool: str, task_type: str
+) -> str:
+    wait_path_url = service_address + "/notebooks/{}/events".format(task_id)
+    wait_page_url = "/det/wait/{}/{}?eventUrl={}&serviceAddr={}".format(
+        task_type, task_id, wait_path_url, service_address
+    )
+    notebook_web_url = "/det/interactive/{}/{}/{}/{}/{}".format(
```
Comment (mapmeld) [inline harness/determined/common/api/request.py:52]: nitpick, but could this variable be changed to `task_web_url` so there's no confusion about it being generic for our other task types

File: harness/determined/common/api/request.py (hunk 1 above)
Comment (mapmeld) [reply harness/determined/common/api/request.py]: From Hamid - the public web URL is an env var https://github.com/hamidzr/determined/blob/5272c09161e3e641d223d5fb1cd97ad4ff0f39c0/webui/react/Makefile#L2

**Comment by julian-determined-ai** [comment]
> Found env var we can use to replace /det/ https://github.com/hamidzr/determined/blob/5272c09161e3e641d223d5fb1cd97ad4ff0f39c0/webui/react/Makefile#L2

Updated! @mapmeld

**Review by mapmeld — APPROVED** [review]
LGTM :+1:

**Pull request merged**